src/reducers/ModalReducer.js

```diff
@@ -2,6 +2,7 @@ import { Map, List } from 'immutable';
 
 const INITIAL_STATE = Map({
     'modalShown': List(['zipcode']),
+    'isPageMaskShown': true,
```

Comment:
is this ever updated?

Reply:
yes, everytime a modal is being toggle, it updates mask at the same time